src/Tests/dotnet-workload-update.Tests/GivenDotnetWorkloadUpdate.cs

```diff
@@ -366,6 +366,8 @@ public void ApplyRollbackAcrossFeatureBand(string existingSdkFeatureBand, string
             packInstaller.InstalledManifests[0].manifestUpdate.ManifestId.Should().Be(manifestsToUpdate[0].manifestUpdate.ManifestId);
             packInstaller.InstalledManifests[0].manifestUpdate.NewVersion.Should().Be(manifestsToUpdate[0].manifestUpdate.NewVersion);
             packInstaller.InstalledManifests[0].manifestUpdate.NewFeatureBand.Should().Be(manifestsToUpdate[0].manifestUpdate.NewFeatureBand);
+            packInstaller.InstalledManifests[0].manifestUpdate.ExistingVersion.Should().Be(manifestsToUpdate[0].manifestUpdate.ExistingVersion);
```

Comment:
Question: Before the change, were these values not set, or did they have different values?

Reply:
These values were introduced in this PR from Daniel: https://github.com/dotnet/sdk/commit/6ba9c46bbdf6989d83a558535487f8ef087dc281
If I remember correctly, Daniel asked me to add these checks because his change was made after the PR where I originally wrote this test